yt_dlp/extractor/mastodon.py

```diff
@@ -0,0 +1,515 @@
+            'thumbnail': 'https://cdn.mastodon.technology/media_attachments/files/006/823/243/small/34571725c91899c8.png',
+            'uploader': '#1 inpost stan',
+            'duration': 12.72,
+            'uploader_url': 'https://mastodon.technology/@BadAtNames',
+        },
+    }, {
+        # pleroma, multiple videos in single post
+        'url': 'https://donotsta.re/notice/9xN1v6yM7WhzE7aIIC',
+        'info_dict': {
+            'id': '9xN1v6yM7WhzE7aIIC@donotsta.re',
+            'uploader_url': 'https://donotsta.re/users/selfisekai',
+            'repost_count': int,
+            'title': '',  # FIXME
+            'uploader_id': 'selfisekai',
+            'comment_count': int,
+            'uploader': 'lauren',
+            'like_count': int,
+        },
+        'playlist': [{
+            'info_dict': {
+                'id': '1264363435@donotsta.re',
+                'ext': 'mp4',
+                'title': 'Cherry Gold💭 - French is one interesting language but this is so funny 🤣🤣🤣🤣-1258667021920845824.mp4',
+                'thumbnail': 'https://donotsta.re/media/888cc71a35dd91acb14d2e180d818c94c34a114d1c9a1f9ebf1365484022a40e.mp4',
+            },
+        }, {
+            'info_dict': {
+                'id': '825092418@donotsta.re',
+                'ext': 'mp4',
+                'title': 'Santi 🇨🇴 - @mhizgoldbedding same guy but i liked this one better-1259242534557167617.mp4',
+                'thumbnail': 'https://donotsta.re/media/4cf3beffd7836b336ab986952c69c2016d0352d50245a2bf1f844efeba194aab.mp4',
+            },
+        }],
+    }, {
+        # pleroma, with /objects/
+        'url': 'https://outerheaven.club/objects/a5046e74-07b4-49a3-9f1c-da11cf97e939',
+        'info_dict': {
+            'id': 'ADbaHO2V0zmsFFjlTM@outerheaven.club',
+            'ext': 'mp4',
+            'uploader': 'Talloran',
+            'comment_count': int,
+            'title': 'ah yes\nthe legendary stealth tactics of sam fisher',
+            'repost_count': int,
+            'like_count': int,
+            'uploader_id': 'Talloran',
+            'thumbnail': 'https://outerheaven.club/media/e478abfb-8dc2-4249-bd0e-81bee9b008b1/Husky_1637445043856_JS39YH64JB.mp4',
+            'uploader_url': 'https://outerheaven.club/users/Talloran',
+        },
+    }, {
+        # mastodon, card to youtube
+        'url': 'https://mstdn.social/@polamatysiak/106183574509332910',
+        'info_dict': {
+            'id': 'RWDU0BjcYp0',
+            'ext': 'mp4',
+            'playable_in_embed': True,
+            'view_count': int,
+            'channel_url': 'https://www.youtube.com/channel/UCLRAd9-Hw6kEI1aPBrSaF9A',
+            'channel': 'Paulina Matysiak',
+            'categories': ['News & Politics'],
+            'live_status': 'not_live',
+            'comment_count': int,
+            'age_limit': 0,
+            'duration': 87,
+            'uploader_url': 'https://mstdn.social/@polamatysiak',  # FIXME: url_transparent?
```

Comment:
Whether we should keep `url_transparent` depends on what we want here. Imo, it's better to not set transparent

Reply:
I don't think it should be `url_transparent`, I don't know where it came from though